forcedotcom/salesforcedx-vscode-automation-tests
# fixes for e2e running on remote - TestSetup.setup()

## What does this PR do?
Adds dynamic wait replacing hard pauses.

## What issues does this PR fix or reference?
@W-13743939@

See also https://salesforce.quip.com/qVEkABuGd7ao and https://docs.google.com/spreadsheets/d/1DwUs_Xb-bGB-1XKZsRurN2l3O6Uxq5n68QizkWRNcSI/edit#gid=0

How to test:
Go to the [End to End Tests Workflow](https://github.com/forcedotcom/salesforcedx-vscode/actions/workflows/e2e.yml) in salesforcedx-vscode repo
Paste the PR's branch (e2e-remote) in the second field of the Run workflow dropdown (Set the branch to use for automation tests)
Choose any e2e test from the dropdown to select the test to run.
AC:
testSetup.setup() should pass.
